.github/workflows/test_build_deb.yml

```diff
@@ -0,0 +1,58 @@
+
+name: Test creation of deb package after code changes
+
+on:
+  push:
+    branches:
+    - develop
+    - 'release-v**'
+  pull_request:
+    branches:
+    - develop
+    - 'release-v**'
+  workflow_dispatch:
+
+  workflow_call:
+    outputs:
+        deb-file-name:
+          description: "DEB file name"
+          value: ${{ jobs.build_deb.outputs.deb-file-name }}
+
+jobs:
+  build_deb:
+    name: Build DEB file
+    runs-on: ubuntu-latest
+    outputs:
+      deb-file-name: ${{ steps.get-deb-filename.outputs.deb-file-name }}
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        repository: learningequality/kolibri-installer-debian
+    - uses: actions/cache@v3
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('setup.py', 'build_requires.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.10.11
+    - name: Download artifact from kolibri repository
```

Comment:
I tend to be a little suspicious of random external github actions. Also, as this is just pulling from the most recent PR build, this could be a little brittle.

If we were going to go this route, we would probably want to add a build on push action on Kolibri, or alternatively a 'nightly' build.

The other option would be to grab it from the latest prerelease assets on Kolibri instead.

Reply:
This is the part of this PR I'm still testing and I'm thinking it's better to remove it here and use a different PR for this as this is not affecting the way this repository is used to create the deb files in new kolibri releases.

The idea with this action was to build a package using the latest kolibri release when changes in this repository happen, to be able to test them. Ideally , any kolibri tar file should be valid to do it. I don't have clear where to fetch it from, or if it's possible for the kolibri release to drop it somewhere in GH where it can be fetched later by a GHA in this repository.  I'm all ears for any suggestion to do it, but in the meantime I'm removing this action from this PR and creating a follow up issue to do this.